packages/compartment-mapper/README.md

```diff
@@ -189,14 +189,6 @@ will never apply.
 If no `exports` apply to the root of the compartment namespace (`"."`),
 the `main` property serves as a default.
 
-> TODO
->
-> The absence of `exports` implies that _all_ of the modules in the package are
-> valid entries.
-> The compartment mapper does not yet support packages that do not name all of
-> their exports in `package.json`, which is unfortunately a significant portion
-> of packages in `npm`.
-
```

Comment:
This is a clean-up. I do not recall when @kumavis or I addressed this problem documented from the original implementation.